theme_enark/views/snippets/s_carousel.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_carousel" inherit_id="website.s_carousel">
```

Comment:
The button is barely visible 😉

| this PR | master |
|--------|--------|
| <img width="1151" alt="image" src="https://github.com/user-attachments/assets/0b231514-1752-41d8-b9a5-c24a0d2cfff0"> | <img width="1029" alt="image" src="https://github.com/user-attachments/assets/fb64b31e-a55f-4761-b039-e1244bcab96d"> |